src/structures/GuildScheduledEvent.js

```diff
@@ -327,6 +337,17 @@ class GuildScheduledEvent extends Base {
   fetchSubscribers(options) {
     return this.guild.scheduledEvents.fetchSubscribers(this.id, options);
   }
+
+  /**
+   * When concatenated with a string, this automatically concatenates the event's URL instead of the object.
+   * @returns {string}
+   * @example
+   * // Logs: Event: https://discord.com/events/412345678901234567/499876543211234567
+   * console.log(`Event: ${guildScheduledEvent}`);
+   */
+  toString() {
+    return this.url;
```

Comment:
These urls don’t get embed when sent so it would probably be better to try and find an invite from the guild’s invite cache to the channel the event is in (if any) and add the event id. If no invites are found then you can default to this URL

Reply:
imho there should be a function to grab both URLs and I would say using `this.url` as the default `toString` allows someone using this library to minimize the permissions of their bot.

If it uses the invite link then the bot _must_ have permissions to create invites.

I'm also optimistically hoping discord will add an embed to this style URL, as the user experience isn't great as-is.

Reply:
Discord cannot add an embed to this url as this doesn’t grant you access to the server the event is on. The toString method also cannot create invites as it cannot be asynchronous so the only possibility would be for it to get from cache or default to the regular url

Reply:
That hybrid behaviour would be a mess anyways. If anything, it should be `fetchEmbed()` if we even want to abstract it.

Reply:
That’d be misleading since it gives you an invite and not an embed object. I think a method called share or something similar would work better and this would create an invite or fetch an existing one

Reply:
tell me why the current behaviour isn't enough. The method does exactly what it should, not getting embedded is on Discord.